src/Compilers/VisualBasic/Portable/CodeGen/EmitConversion.vb

```diff
@@ -230,6 +241,32 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.CodeGen
 
         End Sub
 
+        ''' <summary> 
+        ''' Returns parameterless value type constructor.
+        ''' </summary>
```

Comment:
This should not be normally needed, i think. Either we has "New" in the code and had to bind the ctor, or we had "Nothing" and should not need the ctor. 
This seems to be a preexisting behavior, probably compat, but it seems suspicious.